pages/_app.js

```diff
@@ -1,27 +1,22 @@
 import GlobalStyle from "../styles";
 import { cards } from "../lib/data";
-// import useLocalStorageState from "use-local-storage";
 import { useState, useEffect } from "react";
 import { getCardById } from "../lib/data";
 
 export default function App({ Component, pageProps }) {
   const [id, setId] = useState(undefined);
   useEffect(() => {
-    setId(Math.floor(Math.random() * 77 + 0));
+    setId(Math.floor(Math.random() * 77));
   }, []);
```

Comment:
Is the number 77 the length of an array? I would always use array.length, otherwise there may be problems when the length of the array changes.

I think you don't need the useState at this point.  You never call setId(),except in this place. 
I would use `const id =..`

Reply:
useState combined with useEffect is needed to not get into trouble when rerendering (looking at the random function). i am not saving the id / dailycard to localstorage. dailycard is needed on several sibling pages. 

78 is the size of the array, starting at 0. it is fixed, because this is the size of a traditional tarot deck.

Reply:
Okay, 👌